packages/scripts/src/main/assessCasesMain.ts

```diff
@@ -0,0 +1,99 @@
+import { AzureOpenAI } from "mongodb-rag-core/openai";
+import { assertEnvVars, makeOpenAiEmbedder } from "mongodb-rag-core";
+import { MongoClient } from "mongodb";
+import { Case } from "../Case";
+import { makeSimpleTextGenerator } from "../SimpleTextGenerator";
+import "dotenv/config";
+import { assessRelevance, makeShortName, rateWithLlm } from "../assessCases";
+
+const assessRelevanceMain = async () => {
+  const {
+    FROM_CONNECTION_URI,
+    FROM_DATABASE_NAME,
+    OPENAI_API_KEY,
+    OPENAI_ENDPOINT,
+    OPENAI_API_VERSION,
+    OPENAI_RETRIEVAL_EMBEDDING_DEPLOYMENT,
+    CASE_COLLECTION_NAME,
+  } = assertEnvVars({
+    FROM_CONNECTION_URI: "",
+    FROM_DATABASE_NAME: "",
+    OPENAI_API_KEY: "",
+    OPENAI_ENDPOINT: "",
+    OPENAI_API_VERSION: "",
+    OPENAI_RETRIEVAL_EMBEDDING_DEPLOYMENT: "",
+    CASE_COLLECTION_NAME: "",
+  });
+
+  const openAiClient = new AzureOpenAI({
+    apiKey: OPENAI_API_KEY,
+    endpoint: OPENAI_ENDPOINT,
+    apiVersion: OPENAI_API_VERSION,
+  });
```

Comment:
where is this script to be run? if just from laptop, i recommend using Braintrust proxy. it contains more model options and has caching to make reruns much faster and cheaper.

example usage here: https://github.com/mongodb/chatbot/blob/main/packages/benchmarks/src/discovery/config.ts#L34-L40

Reply:
Ideally we'll run directly from a new page on the Mercury dashboard